calculator/src/main/java/ru/skypro/calculator/ControllerCalc.java

```diff
@@ -0,0 +1,57 @@
+package ru.skypro.calculator;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class ControllerCalc {
+    private final CalculatorImpl calculator;
```

Comment:
```suggestion
    private final CalculatorInterface calculator;
```
инжектить лучше по интерфейсу, чтобы пользоваться полиморфизмом